docs/core-concepts/core-sdk/cli/using-ci-with-cli.md

```diff
@@ -0,0 +1,41 @@
+# Using .NET Core CLI tools in Continuous Integration
```

Comment:
I'd put this in "Scenarios" and would just call it "Using .NET Core with Continuation Integration"